dreammall-earth/dreammall.earth
# feat(frontend): setup sentry for presenter+frontend

Motivation
----------
This will setup Sentry for both our frontends.

We did not add any custom code. That's why no unit test in this PR.

How to test
-----------
1. Set `SENTRY_DSN` in your `frontend/.env`
2. Produce an `auth error` (e.g. not starting `authentik` in `docker compose up`)
3. You will see the error in our `dreammall-frontend` Sentry project

close #2504
